exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/TransferTxTest.java

```diff
@@ -224,7 +205,8 @@ void info() {
     // Check the transaction parameters in JSON
     Gson gson = CryptocurrencyTransactionGson.instance();
 
-    Transaction txParameters = gson.fromJson(info, new TypeToken<TransferTx>() {}.getType());
+    Transaction txParameters = gson.fromJson(info, new TypeToken<TransferTx>() {
+    }.getType());
```

Comment:
This doesn't seem to make it much clearer.

Reply:
it's my autoformat settings - sorry